src/Uno.Toolkit.UI/Controls/SafeArea/SafeArea.cs

```diff
@@ -237,7 +237,7 @@ private static Rect GetVisibleBounds(Rect? safeAreaOverride = null, bool withSof
 				if (inputRect.Top != 0 && inputRect.Top < visibleBounds.Bottom)
 				{
 
-					var windowBottom = XamlWindow.Current.Bounds.Bottom;
+					var windowBottom = XamlWindow.Current?.Bounds.Bottom ?? 0d;
```

Comment:
Does SafeArea correctly handle when XamlWindow.Current is null? And is this only for WinUI, or are there other scenarios where it might be null

Reply:
Ideally it should use `XamlRoot` instead, as we will ultimately make `Current` null across Uno.WinUI (kept it non null to avoid breaking changes for now)

Reply:
Yeah there is still work that needs to be done to make it work with XamlRoot: https://github.com/unoplatform/uno.toolkit.ui/issues/382

This was just a stop-gap to fix the canaries